<?xml version="1.0"?>
<launch>
  <arg name="num_samples" default="1000" />
  <arg name="chain_start" default="r_panda_link0" />
  <arg name="chain_end" default="r_tool" />
  <arg name="timeout" default="0.1" />

  <param name="robot_description" textfile="$(find franka_description)/robots/panda_arm_with_tool.urdf" />

  <node name="rcm_circle" pkg="trac_ik_examples" type="rcm_circle" output="screen">
    <param name="num_samples" value="$(arg num_samples)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="/robot_description"/>
  </node>


</launch>
